inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx

```diff
@@ -251,7 +257,12 @@ export class ToolsTabComponent extends PaneComponent {
 
     createEnvTexture() {
         const scene = this.props.scene;
-        EnvironmentTextureTools.CreateEnvTextureAsync(scene.environmentTexture as CubeTexture)
+        EnvironmentTextureTools.CreateEnvTextureAsync(
+            scene.environmentTexture as CubeTexture,
+            {
+                imageType: envExportImageTypes[this._envOptions.imageType].imageType,
+                imageQuality: this._envOptions.imageQuality

```

Comment:
Related to the first comment, the quality could be part of the envExportImageTypes array ?

Reply:
I don't understand this... `envExportImageTypes` is a const/immutable array defining the options of `<OptionsLineComponent>`, and `imageQuality` is set by the user. So how should those be combined, or what exactly are you suggesting?

Reply:
Yup all good as pngs do not rely on it, and also coming from my previous confusion between the index and the type.